.github/pull_request_template.md

```diff
@@ -1,19 +1,19 @@
-What this PR solves / how to test:
+**What this PR solves / how to test:**
 
-Associated docs issues/PR:
+**Associated docs issue(s)/PR(s):**
 
 - [insert associated docs issue(s)/PR(s)]
 
-Author has included the following, where applicable:
+**Author has included the following, where applicable:**
 
 - [ ] Tests
-- [ ] Changeset
+- [ ] Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))
 
-Reviewer has performed the following, where applicable:
+**Reviewer has performed the following, where applicable:**
 
 - [ ] Checked for inclusion of relevant tests
 - [ ] Checked for inclusion of a relevant changeset
 - [ ] Checked for creation of associated docs updates
 - [ ] Manually pulled down the changes and spot-tested
 
-Fixes # [insert issue number].
+Fixes # [insert GH or internal issue number(s)].
```

Comment:
I think this actually should go right at the top. I was often confused by the "insert associated docs issues/prs" line because I misread it and thought it was asking for the related issue.

Reply:
it would be my preference for it to go higher as well but i remember getting feedback previously that folks wanted it at the bottom. happy to switch it up if there are no objections!

Reply:
+1 to moving it to the top-I often mistakenly put the issue number in the docs section

Reply:
done! :)